### DISCUSSION TITLE
Data link functionalityin Geomap

### DISCUSSION THREAD

**Discussion by HenrikRolandHansen** [grafana/grafana#43124] [Dashboards, Panels & Visualizations]
I would be a nice with a feature to be able to add data links to markers and or hover labels via the override feature. This would make it possible to press the marker or link in the label to "drill down" data  

Example:

If the marker is vessels the marker link would make it possible to jump to dashboard with details like historical positions, speed ,heading ect.

**Comment by ciszkiewicz** [comment]
Second that. There is an option for Data Links but it doesn't work for me

**Comment by nmarrs** [comment]
Hi @HenrikRolandHansen, thanks for the suggestion! It appears that we added data link support in tooltips in [this PR](https://github.com/grafana/grafana/pull/44740)

When you add a datalink it will show up in the tooltip
<img width="862" alt="Screen Shot 2022-10-19 at 2 22 27 PM" src="https://user-images.githubusercontent.com/22381771/196807082-d152ff21-4224-4daf-aab2-d060a84a5f2a.png">


See [data link docs ](https://grafana.com/docs/grafana/v9.0/panels/configure-data-links/) for more info on how to use data links.
